modules/nextMillenniumBidAdapter.js

```diff
@@ -347,15 +355,19 @@ export function setConsentStrings(postBody = {}, bidderRequest) {
 
     if (gdprConsent) {
       if (typeof gdprConsent.gdprApplies !== 'undefined') {
-        postBody.regs.ext.gdpr = gdprConsent.gdprApplies ? 1 : 0;
+        postBody.regs.gdpr = gdprConsent.gdprApplies ? 1 : 0;
       };
 
       if (typeof gdprConsent.consentString !== 'undefined') {
         postBody.user = {
-          ext: { consent: gdprConsent.consentString },
+          consent: gdprConsent.consentString,
         };
       };
     };
+
+    if (config.getConfig('coppa')) {
```

Comment:
Can you get coppa from `bidderRequest.ortb2.regs.coppa` instead please?

Reply:
done